docs/src/inverse_problems/global_sensitivity_analysis.md

```diff
@@ -31,7 +31,7 @@ oprob_base = ODEProblem(seir_model, u0, (0.0, 10000.0), p_dummy)
 function peak_cases(p)
     ps = [:β => p[1], :a => p[2], :γ => p[3]]
     oprob = remake(oprob_base; p = ps)
-    sol = solve(oprob; maxiters = 100000, verbose = false)
+    sol = solve(oprob, Tsit5(); maxiters = 100000, verbose = false)
```

Comment:
Can't you use `OrdinaryDiffEqDefault` to keep the auto-selecting solver?

Reply:
And a similar comment for the other places you switched from the auto-selecting solver to Tsit5.

Reply:
Ah, didn't realize that was an option, will do